numpy/random/_generator.pyx

```diff
@@ -886,9 +886,9 @@ cdef class Generator:
             greater than or equal to low.  The default value is 0.
         high : float or array_like of floats
             Upper boundary of the output interval.  All values generated will be
-            less than high.  The high limit may be included in the returned array of 
-            floats due to floating-point rounding in the equation 
-            ``low + (high-low) * random_sample()``.  high - low must be 
+            less than high.  The high limit may be included in the returned array of
+            floats due to floating-point rounding in the equation
+            ``low + (high-low) * random_sample()``.  high - low must be
```

Comment:
did you mean to make all these whitespace cleanup changes in `_generator.pyx`?

Reply:
Ack, thats a lot of whitespace fixes for one line of changed exception...